### DISCUSSION TITLE
Svelte 5 / runes support?

### DISCUSSION THREAD

**Discussion by tsdexter** [xyflow/xyflow#3994] [Svelte]
Thanks for the excellent work on this. Just wondering if svelte-flow is still in active development and/or if there are plans to move to runes with svelte 5?

**Comment by moklick** [comment]
Thanks! I answered it here: https://github.com/xyflow/xyflow/discussions/3940#discussioncomment-8566733

**Reply by tsdexter** [reply]
Amazing! Thank you - not sure how I missed it, I guess I searched issues for "runes" but not discussions.
